data/json/items/magazine/357.json

```diff
@@ -0,0 +1,18 @@
+[
+	{
+		"id": "p226mag_15rd_357sig",
+		"type": "MAGAZINE",
+		"name": "P226 magazine .357 SIG",
+		"description": "A 15 round double stack box magazine for a SIG P226 chambered for .357 SIG rounds.",
+		"weight": 150,
+		"volume": 1,
+		"price": 3000,
+		"material": "steel",
+		"symbol": "#",
+		"color": "light_gray",
+		"ammo_type": "357sig",
+		"capacity": 15,
+		"reliability": 5,
```

Comment:
What's your source for this? Anything less than 6 ingame sucks as much as improvised firearms.

Reply:
Not sure how that got in there. It was supposed to be 9.